task/09-functions-n-closures-tasks.js

```diff
@@ -26,7 +26,9 @@
  *
  */
```

Comment:
There is no the link to the travis-ci successful build

Reply:
https://travis-ci.org/github/Inquisitor1229/js-assignments/branches

Reply:
ok i'll close this PR
but you had to provide specific link like this https://travis-ci.org/github/Inquisitor1229/js-assignments/builds/707909802